src/_layouts/base.html

```diff
@@ -13,6 +13,11 @@
   </title>
 
   <link href="/css/styles.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;700;900&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@100;400;600;800&display=swap"
+    rel="stylesheet">
```

Comment:
To limit the number of files a user has to download to view your site, you should combine the font stylesheets on lines 17 and 19.